platform/configure/cost-control.mdx

```diff
@@ -167,6 +185,7 @@ platform to manage OpenCost. Therefore, all requests and limits that you wish to
 :::
 
 
-### Cluster Specific Configuration
+### Other cluster settings
+
 For individual cluster configuration, the [Cluster](/platform/api/resources/clusters) resource allows for further
```

Comment:
I believe the api resources need to be updated, since there was additional configuration added to the `Cluster` resource.

Reply:
@hidalgopl do we need to trigger the config update?

Reply:
The PR with a platform partials update should be created automatically on Platform stable release: https://github.com/loft-sh/loft-enterprise/blob/main/.github/workflows/sync-api.yaml#L150-L189

Reply:
Great! @lizardruss if all else looks ok, I'll approve it and merge on release day. Thank you for writing the doc draft!